docker/scripts/maintenance/docker-cleanup.sh

```
@@ -0,0 +1,98 @@
+  log "Removing unused volumes"
+  docker volume prune -f >> "$LOG_FILE" 2>&1
+  
+  # Clean up logs
+  log "Cleaning up Docker logs"
+  if [ -d /var/lib/docker/containers ]; then
+    find /var/lib/docker/containers -type f -name "*.log" -exec truncate -s 0 {} \;
+    log "Docker logs have been truncated"
+  fi
+  
+  # Check disk usage after cleanup
+  local after_usage
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)